### DISCUSSION TITLE
Set textbox's content to a given command's output

### DISCUSSION THREAD

**Discussion by lominoss-git** [davatorium/rofi#1659] [Q&A] [answered]
So I have a textbox whose content/str I want to set to this command's output: ```date +"%I:%M %p"```.
Thanks in advance :)!

**Comment by DaveDavenport** [comment, marked as answer]
See https://github.com/davatorium/rofi/discussions/1637 for similar question + answer.

**Reply by lominoss-git** [reply]
Thank you so much
